DOCUMENTATION.md

```diff
@@ -10,83 +29,86 @@ If you want to read more into Flatseal's permissions, you can look at the [Permi
 --- | --- | --- | ---
-GPU acceleration | Toggle | Allow or prohibit access to the application to graphics direct rendering located at `/dev/dri`. | `--device=dri` and `--nodevice=dri`
-Virtualization | Toggle | Allow or prohibit access to the application to virtualization. | `--device=kvm` and `--nodevice=kvm`
-Shared memory | Toggle | Allow or prohibit access to the application to shared memory. | `--device=shm` and `--nodevice=shm`
-All devices | Toggle | Allow or prohibit access to the application to all devices. | `--device=all` and `--nodevice=all`
+GPU acceleration | Toggle | Allow the application to access the graphics direct rendering located at `/dev/dri` to take advantage of GPU acceleration. | `--device=dri` and `--nodevice=dri`
+Virtualization | Toggle | Allow the application to support virtualization. | `--device=kvm` and `--nodevice=kvm`
+Shared memory | Toggle | Allow the application to access shared memory. | `--device=shm` and `--nodevice=shm`
+All devices | Toggle | Allow the application to access to all devices, such as webcam and external devices. | `--device=all` and `--nodevice=all`
 
 #### Allow
 
 List of features available to the application.
 
 Name | Type | Description | `flatpak override` equivalent
 --- | --- | --- | ---
-Development syscalls | Toggle | Allow or prohibit access to the application to certain syscalls such as [`ptrace()`](https://en.wikipedia.org/wiki/Ptrace) and [`perf_event_open()`](https://en.wikipedia.org/wiki/Perf_(Linux)). | `--allow=devel` and `--disallow=devel`
-Programs from other architectures | Toggle | Allow or prohibit access to the application to execute programs for an [ABI](https://en.wikipedia.org/wiki/Application_binary_interface) other than the one supported natively by the system. | `--allow=multiarch` and `--disallow=multiarch`
-Bluetooth | Toggle | Allow or prohibit access to the application to use bluetooth. | `--allow=bluetooth` and `--disallow=bluetooth`
-Controller Area Network bus | Toggle | Allow or prohibit access to the application to use canbus sockets. You must also have network access for this to work. | `--allow=canbus` and `--disallow=canbus`
+Development syscalls | Toggle | Allow the application to access to certain syscalls, such as [`ptrace()`](https://en.wikipedia.org/wiki/Ptrace) and [`perf_event_open()`](https://en.wikipedia.org/wiki/Perf_(Linux)). | `--allow=devel` and `--disallow=devel`
+Programs from other architectures | Toggle | Allow the application to execute programs for an [ABI](https://en.wikipedia.org/wiki/Application_binary_interface) other than the one supported natively by the system. | `--allow=multiarch` and `--disallow=multiarch`
+Bluetooth | Toggle | Allow the application to use Bluetooth. | `--allow=bluetooth` and `--disallow=bluetooth`
+Controller Area Network bus | Toggle | Allow the application to use canbus sockets. You must also have network access for this to work. | `--allow=canbus` and `--disallow=canbus`
```

Comment:
Add a link to the network access we refer in here.

Reply:
What do you mean? I don't understand

Reply:
I meant, you can link to the actual definition of `network access` you have above.

Reply:
I still don't understand. I don't see where I linked the actual definition of `network access`.

Reply:
Let's rewind... 

In that description you put `You must also have network access for this to work.`. What I am suggesting here is that you you can make the `network access` part of that description an actual link to the `Share` section, so the reader can easily tell what permission you are referring to.

Reply:
https://github.com/tchx84/Flatseal/pull/171/commits/42f5e21f4805c3fda0fd2a2af598639fda7f4370